rush-custom-tips.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
We have several sample nested Rush projects used for unit tests in this project; move this demo file to one of those?

Reply:
I moved it to `rush/-lib/src/logic/test/repo/rush-custom-tips.json`; not sure if that's the right place?